google/cloud/storage/bucket.py

```diff
@@ -2444,6 +2454,9 @@ def generate_signed_url(
         else:
             resource = "/{bucket_name}".format(bucket_name=self.name)
 
+        if use_cname:
+            resource = "/"
```

Comment:
This better be combined with `if` statements upper, as for now you'll construct `resource` variable and then will completely redefine it

Reply:
Yes, i thought the same , like for now don't want to mix-up things with virtual host